components/jumbo/index.tsx

```diff
@@ -44,9 +45,19 @@ export default function Jumbo() {
             />
           </div>
           <div className="container">
-            <Typography className={classes.h1}>
-              {t("helloAlgorithms")}
-            </Typography>
+            <div className={classes.h1}>
+              <Typewriter 
+                  options={{loop:true}}
+                  onInit={(typewriter)=>{
+                    typewriter
+                    .typeString("Hello, algorithms!")
```

Comment:
You should still use the localized string here, so that it also works for other languages.

```suggestion
                    .typeString(t("helloAlgorithms"))
```

Reply:
That was very insightful 😀 I have made required changes. Thankyou